From cb385529e06655c58e3fb360d57a22ec523682e9 Mon Sep 17 00:00:00 2001
From: JP Abgrall <jpa@google.com>
Date: Tue, 5 Mar 2013 14:25:36 -0800
Subject: [PATCH 222/634] input: misc: keychord: log when keychord triggered

log keychord id at info level just before waking up processes.

Signed-off-by: JP Abgrall <jpa@google.com>
---
 drivers/input/misc/keychord.c | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/drivers/input/misc/keychord.c b/drivers/input/misc/keychord.c
index 3ffab6d..a5ea27a 100644
--- a/drivers/input/misc/keychord.c
+++ b/drivers/input/misc/keychord.c
@@ -126,8 +126,12 @@ static void keychord_event(struct input_handle *handle, unsigned int type,
 done:
 	spin_unlock_irqrestore(&kdev->lock, flags);
 
-	if (got_chord)
+	if (got_chord) {
+		pr_info("keychord: got keychord id %d. Any tasks: %d\n",
+			keychord->id,
+			!list_empty_careful(&kdev->waitq.task_list));
 		wake_up_interruptible(&kdev->waitq);
+	}
 }
 
 static int keychord_connect(struct input_handler *handler,
-- 
1.9.1

